contrib/mod_wrap2/mod_wrap2.c

```diff
@@ -362,6 +362,11 @@ static char *wrap2_get_hostname(wrap2_host_t *host) {
       remote_addr->na_have_dnsstr = TRUE;
 
     } else {
+      /* In order to probe the current UseReverseDNS value, we had to enable
+       * earlier.  Make sure we restore that previous value.
+       */
+      pr_netaddr_set_reverse_dns(reverse_dns);
```

Comment:
Consider removing the same call from line 375

Reply:
> Consider removing the same call from line 375

Done!